src/angular-cli.site.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
'site' is ambiguous. Can we rename with angular-cli.default.json or something like that?

Reply:
'default' is too generic. There are 2 scenarios of this repo: wac extension and standalone website. Hence I call the scenario site